docs/reference/setup/install/docker.asciidoc

```diff
@@ -205,24 +140,99 @@ services:
       - esdata02:/usr/share/elasticsearch/data
     networks:
       - esnet
+  es03:
+    image: {docker-image}
+    container_name: es03
+    environment:
+      - node.name=es03
+      - discovery.seed_hosts=es01,es02
+      - cluster.initial_master_nodes=es01,es02,es03
+      - cluster.name=docker-cluster
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - esdata03:/usr/share/elasticsearch/data    
+    networks:
+      - esnet
 
 volumes:
   esdata01:
     driver: local
   esdata02:
     driver: local
+  esdata03:
+    driver: local    
 
 networks:
   esnet:
 --------------------------------------------
 endif::[]
 
-To stop the cluster, type `docker-compose down`. Data volumes will persist,
-so it's possible to start the cluster again with the same data using
-`docker-compose up`.
-To destroy the cluster **and the data volumes**, just type
+To stop the cluster, run `docker-compose down`. 
+The data in the Docker volumes is preserved and loaded when when you restart the cluster with `docker-compose up`.
+To **delete the data volumes** when you bring down the cluster, specify the `-v` option: 
 `docker-compose down -v`.
 
+[[docker-cli-run-prod-mode]]
+==== Using the Docker images in production
+
+[[docker-prod-prerequisites]]
+[IMPORTANT]
+=========================
+
+The `vm.max_map_count` kernel setting needs to be set to at least `262144` for
+production use. Depending on your platform:
+
+* Linux
++
+--
+The `vm.max_map_count` setting should be set permanently in `/etc/sysctl.conf`:
+[source,sh]
+--------------------------------------------
+grep vm.max_map_count /etc/sysctl.conf
+vm.max_map_count=262144
+--------------------------------------------
+
+To apply the setting on a live system type: `sysctl -w vm.max_map_count=262144`
+--
+
+* macOS with https://docs.docker.com/engine/installation/mac/#/docker-for-mac[Docker for Mac]
++
+--
+The `vm.max_map_count` setting must be set within the xhyve virtual machine:
+
+. From the command line, run:
++
+["source","sh"]
+--------------------------------------------
+screen ~/Library/Containers/com.docker.docker/Data/vms/0/tty
+--------------------------------------------
+
+. Press enter and use`sysctl` to configure `vm.max_map_count`:
```

Comment:
After this step, what do you think about having a further step for exiting the `screen` session? I had to enter `Control-a` then `Control-d`.